config/environments/production.rb

```diff
@@ -147,4 +147,5 @@
       end
     }
   end
+  config.collection_resources_s3_bucket_name = "core-prod-collection-resources"
```

Comment:
This (for each env) seems like it should be using an env variable, so that the terraform can just set that up without us needing to keep things in sync if they change for any reason/one of the bucket names can't follow the pattern.